graph/cytoscape/cytoscape.go

```diff
@@ -279,6 +281,31 @@ func addTelemetry(ed *EdgeData, sn *tree.ServiceNode, nd *NodeData, o options.Ve
 	}
 }
 
+func addTLS(ed *EdgeData, sn *tree.ServiceNode, o options.VendorOptions) {
+	if sn.Parent == nil {
+		return
+	}
+
+	nmTLS, ok := sn.Metadata["hasmTLS"].(bool)
+	if !ok {
+		nmTLS = false
+	}
+
+	nMissingSideCars, ok := sn.Metadata["hasMissingSidecars"].(bool)
+	if !ok {
+		nMissingSideCars = true
+	}
+
+	pnMissingSideCars, ok := sn.Parent.Metadata["hasMissingSidecars"].(bool)
+	if !ok {
+		pnMissingSideCars = true
+	}
+
+	if nmTLS && !nMissingSideCars && !pnMissingSideCars {
```

Comment:
I see, you use it for the final calculation, It could be nice to put a comment to understand the logic behind this nmTLS check (someone could think that it could be un-related to sidecars).

Reply:
nmTLS is unrelated to sidecars. nmTLS says is a node has proper DestinationRule enabling TLS.
So the expression: `nmTLS && !nMissingSideCars && !pnMissingSideCars` means that: 
* node has a route rule enabling TLS
* node has istio sidecars
* parent node has istio sidecars

Besides that, nmTLS is computed as all other metadata information appended on each node.